sites/kit.svelte.dev/src/routes/+layout.svelte

```diff
@@ -24,6 +24,9 @@
 				selected={$page.url.pathname.startsWith(`${base}/docs`) || undefined}
 				href="{base}/docs">Docs</NavItem
 			>
+			
+			<NavItem external="https://learn.svelte.dev/tutorial">Learn</NavItem>
```

Comment:
Also, perhaps the link should go to https://learn.svelte.dev/tutorial/introducing-sveltekit ?
```suggestion
			<NavItem external="https://learn.svelte.dev/tutorial/introducing-sveltekit">Tutorial</NavItem>
```